### ISSUE TITLE
Questions about applying ISD loss to training of monovit

### ISSUE THREAD

**Issue by Shaw-Way** [zxcqlf/GasMono#5]
Hi, thanks for your remarkable work.
I noticed that you tested the ISD loss in monovit and got a nice performance improvement. Are the learning rate and number of epochs for training the same as in the monovit paper(5e−5, 20 epochs) or the same as in the gasmono code(1e-4, 40 epochs).
